File: packages/core/test/fixtures/tree-sitter/typescript/launchdarkly/positive/multi-line.ts
import * as LaunchDarkly from 'launchdarkly-node-server-sdk'

const client = LaunchDarkly.init('sdk-key')

export async function multi(user: { key: string }) {
  const result = await client.variation(
    'MULTI_LINE_FLAG',
    user,
    false,
  )
  return result
}

File: packages/core/test/fixtures/tree-sitter/go/launchdarkly/positive/multi-line.go
package main

import (
	ld "github.com/launchdarkly/go-server-sdk/v7"
)

func multi() {
	enabled, _ := client.BoolVariation(
		"GO_MULTI_LINE",
		ldcontext.New("user"),
		false,
	)
	_ = enabled
}
